src/constants/dashboard.js

```diff
@@ -20,6 +20,68 @@ export const incomplete_state = {
     background: "bg-gray-500",
 }
 
+export const activitiesData = [
+  {
+    icon: feed_activity,
+    type: "Comida",
+    time: "4:48:24 PM",
+    info: "400 gr",
+    state: feed_state,
+  },
+  {
+    icon: incomplete_activity,
+    type: "-",
+    time: "-",
+    info: "-",
+    state: incomplete_state,
+  },
+  {
+    icon: incomplete_activity,
+    type: "-",
+    time: "-",
+    info: "-",
+    state: incomplete_state,
+  },
+  {
+    icon: incomplete_activity,
+    type: "-",
+    time: "-",
+    info: "-",
+    state: incomplete_state,
+  },
+];
+
+export const noPetsActivitiesData = [
+  {
+    icon: incomplete_activity,
+    type: "-",
+    time: "-",
+    info: "-",
+    state: incomplete_state,
+  },
+  {
+    icon: incomplete_activity,
+    type: "-",
+    time: "-",
+    info: "-",
+    state: incomplete_state,
+  },
+  {
+    icon: incomplete_activity,
+    type: "-",
+    time: "-",
+    info: "-",
+    state: incomplete_state,
+  },
+  {
+    icon: incomplete_activity,
+    type: "-",
+    time: "-",
+    info: "-",
+    state: incomplete_state,
+  },
+];
+
```

Comment:
Esto se puede refactorizar un poco
```suggestion
const incompleteActivity = {
    icon: incomplete_activity,
    type: "-",
    time: "-",
    info: "-",
    state: incomplete_state,
}

export const activitiesData = [
  {
    icon: feed_activity,
    type: "Comida",
    time: "4:48:24 PM",
    info: "400 gr",
    state: feed_state,
  },
  incompleteActivity,
  incompleteActivity,
  incompleteActivity,
];

export const noPetsActivitiesData = [
  incompleteActivity,
  incompleteActivity,
  incompleteActivity,
  incompleteActivity,
];

```